op-node/rollup/derive/pipeline.go

```diff
@@ -209,12 +209,12 @@ func (dp *DerivationPipeline) Step(ctx context.Context) error {
 			return nil
 		}
 	}
-
 	// Now step the engine queue. It will pull earlier data as needed.
 	if err := dp.eng.Step(ctx); err == io.EOF {
 		// If every stage has returned io.EOF, try to advance the L1 Origin
 		return dp.traversal.AdvanceL1Block(ctx)
-	} else if errors.Is(err, EngineP2PSyncing) {
+	} else if errors.Is(err, EngineELSyncing) {
+		// Log that we are still waiting for EL to sync?
 		return err
```

Comment:
I wouldn't log here given we're returning an error. Plus logging here is likely to generate too much noise.